shared/src/business/utilities/DateHandler.js

```diff
@@ -443,6 +445,43 @@ const subtractISODates = (date, dateConfig) => {
   return DateTime.fromISO(date).minus(dateConfig).setZone('UTC').toISO();
 };
 
+/**
+ * Returns startDate plus n numberOfDays
+ * but if the return date results on a Saturday, Sunday, or Federal holiday,
+ * it will be moved forward to the next business day
+ *
+ * @param {string} startDate the date to add days to
+ * @param {number} numberOfDays number of days to add to startDate
+ * @returns {string} a formatted MONTH_DAY_YEAR string if date object is valid
+ */
+const getDateInFuture = ({ numberOfDays, startDate, units = 'days' }) => {
```

Comment:
Should this perhaps be named getBusinessDateInFuture so it's a little more clear that it's not returning weekends or holidays?

Reply:
good idea!